jobs/webhooks/templates/config/kubeconfig.yaml.erb

```diff
@@ -0,0 +1,23 @@
+<%
```

Comment:
Duplicate of apiserver kubeconfig job.......can remove it, as there would be never a case of webhook without apiserver. If you still want to have it, we can re-use the code atleast.